ui/app/pages/settings/contact-list-tab/index.scss

```diff
@@ -191,10 +187,8 @@
     }
 
     &__text {
-      font-style: normal;
-      font-weight: normal;
-      font-size: 14px;
-      line-height: 20px;
+      @include Paragraph;
```

Comment:
Believe we should be using font-size 16 for text, but this could also be a H6

<details>
<summary>Before</summary>

<img src="https://user-images.githubusercontent.com/4448075/88848449-1d0b0e80-d1ae-11ea-9cf4-e069006dc01c.png" />



</details>

<details>
<summary>After</summary>

<img src="https://user-images.githubusercontent.com/4448075/88848369-fea51300-d1ad-11ea-82b6-9c342f25138b.png" />


</details>

Reply:
Both seem OK to me! @rachelcope , thoughts?

Reply:
I think in the designs we've moved towards font-size 14 for subheader text. The font-size 16 looks a little big here. Is the default paragraph styles font-size 16?

Reply:
@rachelcope paragraph styles match the figma design system at 1REM / 16px. Is this a subtitle or text? Cause its labeled as text, but happens right under a title so could be a subtitle too lol